boards/xtensa/esp32s3/common/Kconfig

```diff
@@ -80,6 +80,159 @@ endchoice # ESP32S3_SPIFLASH_FS
 
 if PM
 
+
+config PM_EXT1_WAKEUP
+	bool "PM EXT1 Wakeup"
+	default n
+	---help---
+		Enable EXT1 wakeup functionality.
+		This allows the system to wake up from PM_STANDBY or PM_SLEEP
+		when a GPIO pin configured as an EXT1 wakeup source is triggered.
+
+if PM_EXT1_WAKEUP
+
+	menu "PM EXT1 Wakeup Sources"
+
+	config PM_EXT1_WAKEUP_RTC_GPIO0
+		bool "RTC_GPIO0"
+		default n
+		help
+			Enable RTC GPIO0 as an EXT1 wakeup source.
+
```

Comment:
Hi @thiagofinelon , regarding the style of the NuttX Kconfig file:

menu, config  should not have any tabs or spaces on the left.

help -> ---help---

bool, default, string , ---help---  one tab on the left.

description below ---help---  two tabs on the left.